lua/lazy/manage/rocks.lua

```diff
@@ -0,0 +1,75 @@
+local Config = require("lazy.core.config")
+local Util = require("lazy.core.util")
+
+---@class LazyRock
+---@field plugin string
+---@field name string
+---@field spec string
+---@field installed boolean
+
+local M = {}
+---@type LazyRock[]
+M.rocks = {}
+
+---@param ... string
+---@return string[]
+function M.args(...)
+  local ret = { "--tree", Config.rocks.tree, "--lua-version", "5.1" }
```

Comment:
suggestion: a `Config.rocks.servers` option could allow adding custom servers like `--server='https://nvim-neorocks.github.io/rocks-binaries/'` (for installing pre-built binary rocks like `toml-edit`)

Reply:
Great idea. Added!